cmd/buck/cli/archive.go

```diff
@@ -1,18 +1,74 @@
 package cli
 
 import (
+	"bytes"
 	"context"
+	"encoding/json"
+	"io"
+	"os"
 
 	"github.com/manifoldco/promptui"
 	"github.com/spf13/cobra"
 	"github.com/textileio/textile/v2/buckets/local"
 	"github.com/textileio/textile/v2/cmd"
 )
 
+var defaultArchiveConfigCmd = &cobra.Command{
```

Comment:
Just noticing now that these archive commands (the older ones as well) as missing the `Args` param, which is handy to restrict args, e.g., `Args: cobra.ExactArgs(0),`, `Args: cobra.RangeArgs(1, 2),`, `Args:  cobra.MaximumNArgs(1),`, etc. It also shows in the `--help` output.